### PR TITLE
Create KSPScienceRedone.netkan

### PR DESCRIPTION
**Pull request by ValynEritai**
Adding a NETKAN for KSPScienceRedone.

### CODE REVIEW

File: NetKAN/KSPScienceRedone.netkan [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "KSPScienceRedone",
+    "$kref"        : "#/ckan/github/ValynEritai/KSPScienceRedone",
```
Comment (politas) [inline NetKAN/KSPScienceRedone.netkan]: This needs to match the GitHub Repository name:

    "$kref"        : "#/ckan/github/ValynEritai/ksp-science-redone",

Comment (Olympic1) [inline NetKAN/KSPScienceRedone.netkan]: pinging @ValynEritai

**Comment by ValynEritai** [comment]
Sorry about not getting to this sooner, I've had no PC since the end of March and only just got it back.

**Comment by Olympic1** [comment]
No problem, it is indexed now

**Pull request closed without merging**